validator/client/src/test/java/tech/pegasys/teku/validator/client/AbstractDutySchedulerTest.java

```diff
@@ -44,8 +45,10 @@
   final ValidatorIndexProvider validatorIndexProvider = mock(ValidatorIndexProvider.class);
   final Signer validator1Signer = mock(Signer.class);
   final Signer validator2Signer = mock(Signer.class);
-  final Validator validator1 = new Validator(VALIDATOR1_KEY, validator1Signer, Optional.empty());
-  final Validator validator2 = new Validator(VALIDATOR2_KEY, validator2Signer, Optional.empty());
+  final Validator validator1 =
+      new Validator(VALIDATOR1_KEY, validator1Signer, new FileBackedGraffitiProvider(Optional.empty(), Optional.empty()));
```

Comment:
(nit) For these tests, I'd just use a simple lambda to return the empty value
```suggestion
      new Validator(VALIDATOR1_KEY, validator1Signer, () -> Optional.empty());
```